mslib/mscolab/server.py

```diff
@@ -346,15 +347,21 @@ def message_attachment():
     if file is not None:
         with fs.open_fs('/') as home_fs:
             file_dir = fs.path.join(APP.config['UPLOAD_FOLDER'], op_id)
-            if not home_fs.exists(file_dir):
-                home_fs.makedirs(file_dir)
+            if '\\' not in file_dir:
```

Comment:
What is the issue here?
Is this really the state of the art for cross-platform software development?

Reply:
python escape by windows path seperator

There were only a few places left where these lines were missing. At least now are all marked to get solved same way.